src/main/java/me/mrCookieSlime/Slimefun/api/BlockStorage.java

```diff
@@ -234,9 +242,15 @@ private void loadChunks() {
 
     private void loadInventories() {
         for (File file : new File("data-storage/Slimefun/stored-inventories").listFiles()) {
-            if (file.getName().startsWith(world.getName()) && file.getName().endsWith(".sfi")) {
+            if (file.getName().endsWith(".sfi")) {
```

Comment:
Why not just change this to a .equals rather than removing the startsWith and slowing loading down?

Reply:
because its in the format world;x;y;z.sfi

Reply:
God this stuff is horrible...

Hmm. I need to look into how this stuff is working a bit